code/src/components/MoviesList.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { LIST_URL } from '../data/Url';
+import '../css/MoviesList.css'
+
+export const MoviesList = () => {
+  const [movies, setMovies] = useState([])
+  const [loading, setLoading] = useState(false)
+
+  useEffect(() => {
+    setLoading(true)
+    fetch(LIST_URL)
+      .then((res) => res.json())
+      .then((json) => {
+        setMovies(json.results)
+        setTimeout(() => setLoading(false), 200)
+      })
+  }, [])
+  console.log(movies)
+
+  if (loading) {
+    return <p className="loader">Loading</p>
```

Comment:
When the page is loading it also shows the footer high up on the page which, so maybe the footer needs to be in a fixed position at the bottom of the page?